packages/apollo/src/config.ts

```diff
@@ -232,7 +238,10 @@ export async function resolveSchema(
       )
     : referredSchema.clientSide
       ? buildASTSchema(loadAsAST())
-      : await loadSchema(referredSchema, config);
+      : await loadSchema(referredSchema, config).then(introspectionResult => {
```

Comment:
I think `introspectionResult` is actually `IntrospectionSchema`, so maybe the name should reflect that.

Reply:
Just saw this!  Fixed via 8e144a1120a63ae6a1aecfb73231b64eddc3f668.

Reply:
@martijnwalraven great catch on that! That difference (between `IntrospectionSchema` and `IntrospectionQuery` tripped me up a few times). Thanks @abernix for the change!